README.md

```diff
@@ -1,2 +1,16 @@
 # web-scraper-helper
 A tool to test web scraping rules.
+
+## Description
+
+This tool is a Tampermonkey script. You need to install the [Tampermonkey](https://chrome.google.com/webstore/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo) extension in your Chrome browser to use it.
+
+## How-to run
+
+* Import the `misc/Webscraping.user.js` file in your Tampermonkey extension.
+* From within the Tampermonkey editor:
+  * Edit the `jsonRules`
+  * Edit the `@match` to include the site you are working with
+  * Save the script
+* Reload your site
+
```

Comment:
We should have a dependency section stating that you need temperMonkey installed in your browser in order to run,